tests/unit/FirmsAddressList.spec.ts

```diff
@@ -163,12 +163,16 @@ describe('FirmsAddressList', () => {
     wrapper.destroy()
   })
 
-  it('displays "Not Enter" for null address', async () => {
+  it('displays "(Not Entered)" for null address', async () => {
     // init store
     store.state.businessAddress = {
       deliveryAddress: null,
       mailingAddress: null
     }
+    store.state.getBusinessAddress = {
```

Comment:
`getBusinessAddress` is not a state property.

Are you trying to override a store getter? See https://vuex.vuejs.org/guide/getters.html#getters

...Or you could mock it.

...Or, why not set the state the way you want it on lines 168-170?

Reply:
I will create another test case for getBusinessAddress and use adifferent state.businessAddress.